ragna/deploy/_ui/api_wrapper.py

```diff
@@ -18,7 +18,7 @@ class ApiWrapper(param.Parameterized):
     auth_token = param.String(default=None)
 
     def __init__(self, api_url, **params):
-        self.client = httpx.AsyncClient(base_url=api_url)
+        self.client = httpx.AsyncClient(base_url=api_url, timeout=60)
```

Comment:
Even when streaming, the Google assistants return really large chunks and thus easily go over the default timeout. The new timeout is in line with what we use for our builtin assistants as well:

https://github.com/Quansight/ragna/blob/1b53e62cb873ba16968b7c25a81baff0947d705c/ragna/assistants/_api.py#L19-L22